<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProgressDialog</class>
 <widget class="QDialog" name="ProgressDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel
{
	color: rgb(84, 84, 84);
	font: 14pt &quot;Segoe UI&quot;;
}
QPushButton
{
	color: rgb(84, 84, 84);
	font: 14pt &quot;Segoe UI&quot;;
}
QWidget
{
	background-color: rgb(255, 255, 255);
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="0">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="1">
       <widget class="QPushButton" name="btnBack">
        <property name="styleSheet">
         <string notr="true">QPushButton
{
  border-image: url(&quot;:/resource/audiocd-popup-btn110-ok-n.png&quot;);
width: 110px;
color: rgb(84, 84, 84)
}

QPushButton:hover
{
  border-image: url(&quot;:/resource/audiocd-popup-btn110-ok-h.png&quot;);
width: 110px;
color: rgb(255, 255, 255)
}</string>
        </property>
        <property name="text">
         <string>Background</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="4">
       <widget class="QPushButton" name="btnClose">
        <property name="styleSheet">
         <string notr="true">QPushButton
{
  border-image: url(&quot;:/resource/audiocd-popup-btn110-ok-n.png&quot;);
width: 110px;
color: rgb(84, 84, 84)
}

QPushButton:hover
{
  border-image: url(&quot;:/resource/audiocd-popup-btn110-ok-h.png&quot;);
width: 110px;
color: rgb(255, 255, 255)
}</string>
        </property>
        <property name="text">
         <string>Close</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="btnSkip">
        <property name="styleSheet">
         <string notr="true">QPushButton
{
  border-image: url(&quot;:/resource/audiocd-popup-btn110-ok-n.png&quot;);
width: 110px;
color: rgb(84, 84, 84)
}

QPushButton:hover
{
  border-image: url(&quot;:/resource/audiocd-popup-btn110-ok-h.png&quot;);
width: 110px;
color: rgb(255, 255, 255)
}</string>
        </property>
        <property name="text">
         <string>Skip</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="btnStop">
        <property name="styleSheet">
         <string notr="true">QPushButton
{
  border-image: url(&quot;:/resource/audiocd-popup-btn110-ok-n.png&quot;);
width: 110px;
color: rgb(84, 84, 84)
}

QPushButton:hover
{
  border-image: url(&quot;:/resource/audiocd-popup-btn110-ok-h.png&quot;);
width: 110px;
color: rgb(255, 255, 255)
}</string>
        </property>
        <property name="text">
         <string>Stop</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QFrame" name="frame_3">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="labelCurTime">
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelTotTime">
        <property name="text">
         <string>-</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>520</width>
        <height>128</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QLabel" name="labelContent">
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QFrame" name="frame_2">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="1" column="0">
       <widget class="QProgressBar" name="progressBar">
        <property name="value">
         <number>24</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
